Support custom steps entrypoint (#2985)
woodpecker-ci/woodpecker: docs/docs/20-usage/20-workflow-syntax.md

> Please note that only build steps can define commands. You cannot use commands with plugins or services.

### `entrypoint`

> Allows you to specify the entrypoint for Docker and Kubernetes. Note that this must include the full command as list, including arguments (e.g. `["/bin/sh", "-c"]`).

Reviewer: ```suggestion
Allows you to specify the entrypoint for containers. Note that this must be a list of the command and its arguments (e.g. `["/bin/sh", "-c"]`).
```
